app/src/components/HomePage/Hero.tsx

```diff
@@ -0,0 +1,270 @@
+// only render map on the client
+import dynamic from "next/dynamic";
+import { Box, Heading, Icon, Spinner, Text, Tooltip } from "@chakra-ui/react";
+import { CircleFlag } from "react-circle-flags";
+import { InventorySelect } from "@/components/InventorySelect";
+import { MdArrowOutward, MdGroup, MdOutlineAspectRatio } from "react-icons/md";
+import { InfoOutlineIcon } from "@chakra-ui/icons";
+import { Trans } from "react-i18next/TransWithoutContext";
+import { getShortenNumberUnit, shortenNumber } from "@/util/helpers";
+import type { TFunction } from "i18next";
+import type { PopulationAttributes } from "@/models/Population";
+import type { InventoryResponse } from "@/util/types";
+
+const CityMap = dynamic(() => import("@/components/CityMap"), { ssr: false });
```

Comment:
We might not need to do this any more as we are now using Pigeon Maps instead of Leaflet, which is much more React compatible.

Reply:
As this was just a refactor, I don't want to mess with existing working code too much